### PR TITLE
Editorial mdlint and vale updates for the HowTo/Get-Started directory

### PR DESCRIPTION
**Pull request by grantnoble**
The usual mdlint and vale updates including line length <= 100, EOL spaces removed, appropriate blank lines, passive voice to active voice.

### CODE REVIEW

File: docs/HowTo/Get-Started/Run-Docker-Image.md [hunk 1]
```diff
@@ -1,108 +1,127 @@
...
+* [Docker](https://docs.docker.com/install/)
 
-* MacOS or Linux 
-    
-    !!! important 
-        The Docker image does not run on Windows. 
+* MacOS or Linux
 
-## Default Node for Mainnet
+    !!! important
```
Comment (NicolasMassart) [inline docs/HowTo/Get-Started/Run-Docker-Image.md]: I don't think you need this to be indented. The `!!!` can begin at the start of the line without spaces.

File: docs/HowTo/Get-Started/System-Requirements.md [hunk 2]
```diff
@@ -1,64 +1,69 @@
...
 
-* Memory Size: Set to 4096 (recommended)
+* Ensure you enable Intel Virtualization Technology (VTx) and Virtualization Technology for
+  Directed I/O (VT-d) in the BIOS settings.
+* On Windows, you might need to disable Hyper-V in the Windows Feature list.
 
-* Create a virtual hard disk with at least 10 GB; 20 GB is recommended
+We recommend you create a VM with the following attributes:
 
+* Memory Size: Set to 4096 (recommended)
```
Comment (NicolasMassart) [inline docs/HowTo/Get-Started/System-Requirements.md]: Could you add the unit? 4096 **MB**
MB is for Mega Bytes, should all be uppercased and with a space between the value and the unit.

File: docs/HowTo/Get-Started/Run-Docker-Image.md (hunk 1 above)
Comment (grantnoble) [reply docs/HowTo/Get-Started/Run-Docker-Image.md]: Done.

File: docs/HowTo/Get-Started/System-Requirements.md (hunk 2 above)
Comment (grantnoble) [reply docs/HowTo/Get-Started/System-Requirements.md]: Done.

**Pull request merged**